cookbooks/ey-base/recipes/custom.rb

```diff
@@ -0,0 +1,7 @@
+if fetch_env_var(node, "EY_REDIS_ENABLED")
+  include_recipe 'redis'
+end
+
+if fetch_env_var(node, "EY_MEMCACHED_ENABLED")
```

Comment:
Like in the Redis case, it should be checked for `/^TRUE$/i` explicitly. Maybe introduce another helper, a la `env_var_set_true?`.
Further, this pattern will repeat a lot, so maybe a helper `include_recipe_if_enabled('component-recipe', 'COMPONENT')` makes sense even?

Reply:
Added /^TRUE$/i for redis and rebased on FB-830-env-vars-memcached. Ended up with a modified commit and force pushed.

I'll try using include_recipe_if_enabled('component-recipe', 'COMPONENT').